bentoml/yatai/yatai_service.py

```diff
@@ -143,6 +166,7 @@ def start_yatai_service_grpc_server(
         f'''* Web UI: {f"running on http://127.0.0.1:{ui_port}/{base_url}"
         if (with_ui and base_url!=".")
         else f"running on http://127.0.0.1:{ui_port}" if with_ui else "off"}\n'''
+        f'* Prometheus running on 127.0.0.1:{prometheus_port}\n'
```

Comment:
@aarnphm why is this running on a separate port?  Is there a way to expose it on the web UI server? e.g. `http://127.0.0.1:{ui_port}/metrics`